extract-metadata.py

```diff
@@ -44,7 +70,10 @@ def processingInstruction(self, target, data):
         fileext = os.path.splitext(filename)[1]
         extract_func = extract_funcs.get(fileext, lambda a: {})
         frontmatter = extract_func(filename)
-        value = frontmatter.get(target, "")
+        if target == "title" and frontmatter.get("abbrev", None) != None:
+            value = frontmatter["abbrev"]
```

Comment:
Wouldn't it make sense to query for abbrev if you want the abbreviated name?

Then abbrev has a special fallback to title, as follows:

```python
if target == "abbrev":
    value = frontmatter.get("abbrev", frontmatter.get("title", ""))
else:
    value = frontmatter.get(target, "")
```

Reply:
```suggestion
            value = frontmatter.get("abbrev", frontmatter.get("title", ""))
```